app/src/full/java/org/openobservatory/ooniprobe/common/FirebaseMessagingService.java

```diff
@@ -5,32 +5,60 @@
 
 import com.google.firebase.messaging.RemoteMessage;
 
-import org.json.JSONObject;
+import org.openobservatory.ooniprobe.R;
+import org.openobservatory.ooniprobe.activity.OoniRunActivity;
 
-import java.util.Map;
+import ly.count.android.sdk.messaging.CountlyPush;
 
 public class FirebaseMessagingService extends com.google.firebase.messaging.FirebaseMessagingService {
 	private static final String TAG = "FCM";
 
 	@Override
 	public void onMessageReceived(RemoteMessage remoteMessage) {
-		// TODO(developer): Handle FCM messages here.
-		// Not getting messages here? See why this may be: https://goo.gl/39bRNJ
-		Log.d(TAG, "Message from: " + remoteMessage.getFrom());
-		Log.d(TAG, "Message data payload: " + remoteMessage.getData());
-		Map<String, String> params = remoteMessage.getData();
-		// Check if message contains a data payload.
-		if (remoteMessage.getData().size() > 0) {
-			try {
-				//TODO-FUTURE we can use click_action instead of type
-				JSONObject data = new JSONObject(params.toString());
-			} catch (Exception e) {
-				System.out.println("JSONException " + e);
-			}
+		super.onMessageReceived(remoteMessage);
+
+		Log.d("DemoFirebaseService", "got new message: " + remoteMessage.getData());
```

Comment:
Please change the string with which we identify to something different from `DemoFirebaseService`?

Reply:
That's right, I need a tag